res/http.php

```diff
@@ -27,8 +30,11 @@
  * HTTP stack.
  */
 return pipe([
+    BlackfireMiddleware::class,
+    NewRelicMiddleware::class,
     MaintenanceMiddleware::class,
     ErrorHandlerMiddleware::class,
+    AssetsMiddleware::class,
```

Comment:
This middleware is useful only in dev environment. It serves static files (aka assets) like CSS, JS, etc.

Previously we didn't need it because PHP's built-in webserver (`php -S 127.0.0.1:8000`) was handling it automatically. But since we are using React as the webserver, we need to serve the static files.

Using a middleware is not really efficient performance-wise, but this middleware is only serving files in `dev` environment, not in production.

In production the middleware will not do much work, still it could be optimized. However I don't really care, the application is super duper fast, that's maybe something to optimize later if we really care.